### PR TITLE
5 packages from Psi-Prod/Mehari at 0.1

### PR DESCRIPTION
**Pull request by ArtichOwO**
This pull-request concerns:
- `mehari.0.1`: A cross-platform library for building Gemini servers
- `mehari-eio.0.1`: Mehari IO implementation using Eio
- `mehari-eio-unix.0.1`: Mehari IO implementation using Eio and Unix bindings
- `mehari-lwt-unix.0.1`: Mehari IO implementation using Lwt and Unix bindings
- `mehari-mirage.0.1`: Mehari IO implementation for MirageOS

---
* Homepage: https://github.com/Psi-Prod/Mehari
* Source repo: git+https://github.com/Psi-Prod/Mehari.git
* Bug tracker: https://github.com/Psi-Prod/Mehari/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

File: packages/mehari-lwt-unix/mehari-lwt-unix.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+synopsis: "Mehari IO implementation using Lwt and Unix bindings"
+maintainer: ["tim.arnouts@protonmail.com" "lelolartichaut@laposte.net"]
+authors: ["The mehari programmers"]
+license: "LGPL-3.0-or-later"
+homepage: "https://github.com/Psi-Prod/Mehari"
+bug-reports: "https://github.com/Psi-Prod/Mehari/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "mehari"
+  "mehari-mirage"
```
Comment (kit-ty-kate) [inline packages/mehari-lwt-unix/mehari-lwt-unix.0.1/opam:12]:
```suggestion
  "mehari" {= version}
  "mehari-mirage" {= version}
```

File: packages/mehari-eio/mehari-eio.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+synopsis: "Mehari IO implementation using Eio"
+maintainer: ["tim.arnouts@protonmail.com" "lelolartichaut@laposte.net"]
+authors: ["The mehari programmers"]
+license: "LGPL-3.0-or-later"
+homepage: "https://github.com/Psi-Prod/Mehari"
+bug-reports: "https://github.com/Psi-Prod/Mehari/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "ocaml" {>= "5.0.0"}
+  "mehari"
```
Comment (kit-ty-kate) [inline packages/mehari-eio/mehari-eio.0.1/opam:12]:
```suggestion
  "ocaml" {>= "5.0.0"}
  "mehari" {= version}
```

File: packages/mehari-eio-unix/mehari-eio-unix.0.1/opam [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+maintainer: ["tim.arnouts@protonmail.com" "lelolartichaut@laposte.net"]
+authors: ["The mehari programmers"]
+license: "LGPL-3.0-or-later"
+homepage: "https://github.com/Psi-Prod/Mehari"
+bug-reports: "https://github.com/Psi-Prod/Mehari/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "ocaml" {>= "5.0.0"}
+  "mehari"
+  "mehari-eio"
```
Comment (kit-ty-kate) [inline packages/mehari-eio-unix/mehari-eio-unix.0.1/opam:13]:
```suggestion
  "mehari" {= version}
  "mehari-eio" {= version}
```

**Review by kit-ty-kate — COMMENTED** [review]
Unless you do not plan to release all those packages together each release, adding `{= version}` to make them sync up is a good idea.

**Comment by ArtichOwO** [comment]
Thanks for your suggestions, I commited it!

**Comment by kit-ty-kate** [comment]
Thanks! Could you return the fixes to the opam files upstream so that they can be taken into account for next time?

**Comment by ArtichOwO** [comment]
Done, thanks for merging!

**Pull request merged**